### PR TITLE
Migration to monorepo

### PR DESCRIPTION
**Pull request by literat**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 LMC s.r.o.
```
Comment (crishpeen) [inline LICENSE.md:3]:
```suggestion
Copyright (c) 2022 LMC s.r.o.
```

**Review by crishpeen — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,23 +1,50 @@
-# Jobs Design Tokens
+<div align="center">
+<img src="https://github.com/lmc-eu/jobs-design-system/blob/main/static/jobs.svg?raw=true" width="422" height="300" alt="Jobs Design System" />
```
Comment (crishpeen) [inline README.md:2]: I just noticed, that this doesn't exist, right?

File: LICENSE.md (hunk 1 above)
Comment (literat) [reply LICENSE.md]: Fixed.

File: README.md (hunk 2 above)
Comment (literat) [reply README.md]: Removed.

**Pull request merged**